orch-configs/profiles/enable-lenovo.yaml

```diff
@@ -6,6 +6,9 @@ argo:
   infra-managers:
     onboarding-manager:
       enabled: false
+    os-resource-manager:
+      enabledProfiles:
+        - ubuntu-lenovo
```

Comment:
two doubts:
- do we need to keep this here?
- if yes - shall we move the `disableProviderAutomation` too?